File: li/cil/oc/api/OpenComputersAPI.java
package li.cil.oc.api;

import li.cil.oc.server.computer.Drivers;

public class OpenComputersAPI {
  static void addDriver(IBlockDriver driver) {
    // TODO Use reflection to allow distributing the API.
    Drivers.add(driver);
  }

  static void addDriver(IItemDriver driver) {
    // TODO Use reflection to allow distributing the API.
    Drivers.add(driver);
  }
}

File: li/cil/oc/api/scala/OpenComputersAPI.scala
package li.cil.oc.api.scala

import li.cil.oc.server.computer.Drivers

object OpenComputersAPI {
  def addDriver(driver: IBlockDriver) {
    // TODO Use reflection to allow distributing the API.
    Drivers.add(driver)
  }

  def addDriver(driver: IItemDriver) {
    // TODO Use reflection to allow distributing the API.
    Drivers.add(driver)
  }
}